sympy/integrals/integrals.py

```diff
@@ -832,6 +834,24 @@ def _eval_integral(self, f, x, meijerg=None, risch=None, manual=None,
             if not meijerg:
                 # g(x) = Mul(trig)
                 h = trigintegrate(g, x, conds=conds)
+                if isinstance(f,Sum):
+                    f_temp =f
+                    f, limits = f.function, list(f.limits)
+                    limit = limits.pop(-1)
```

Comment:
Aren't you overwriting the already imported `limit` function? This is a bad practise to overwrite.